server.js

```diff
@@ -81,8 +81,51 @@ server.route({
+                var issueBody = `### 🆕🐥☝ First Timers Only.
 
+This issue is reserved for people who never contributed to Open Source before. We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝 [About First Timers Only](http://www.firsttimersonly.com/).
+
+### 🤔 What you will need to know.
+
+Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.
+
+### :clipboard: Step by Step
+
+- [ ] 🙋 **Claim this issue**: Comment below.
+
+  Once claimed we add you as contributor to this repository.
+
+- [ ] 👌 **Accept our invitation** to this repository. Once accepted, assign yourself to this repository
+
+- [ ] 👓 **Please review** our [Code of Conduct](http://hood.ie/code-of-conduct/)
+
+  In a nutshell: be patient and actively kind
+
+- [ ] 🔄 **replace** the \`up for grabs\` label with \`in progress\`.
+
+- [ ] 📝 **Update** the file [${fileName}](${blobURL}) in the \`hoodie\` repository (press the little pen Icon) and edit the line as shown below.
+
+  \`\`\`diff
+  ${patchDiff.replace(/\n/g, '\n  ')}
+  \`\`\`
+
+- [ ] 💾 **Commit** your changes
+
+- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:
+
+1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.
+
+2. You can [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/)
+
+- [ ] 🏁 **Done** :+1: Replace the \`in progress\` label with \`ready\`. Ask in comments for a review :)
+
+### 🤔❓ Questions
+
+You can ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/)`
+
+                console.log(`\nissueBody ==============================`)
+                console.log(issueBody)
+                return
```

Comment:
ugh sorry I left this in for debugging myself, can you please remove it? Again my fault, I’m sorry :(